marklogic-data-hub/src/test/ml-modules/root/test/suites/data-hub/5/impl/hub-entities/generateTDE.sjs

```diff
@@ -0,0 +1,92 @@
+const hent = require("/data-hub/5/impl/hub-entities.xqy");
+const test = require("/test/test-helper.xqy");
+
+function generateTdeWithViewJoin() {
+    const input =
```

Comment:
Can you slim this down to the minimum needed to reproduce the problem? I'm guessing just the customerId and billing, and then you can drop Zip from Address? Or is Zip needed?

Reply:
I adjusted the test to also check the single value join of address to zip.